ixp_tracker/models.py

```diff
@@ -9,6 +9,7 @@ class IXP(models.Model):
     website = models.URLField(null=True)
     active_status = models.BooleanField(default=True)
     manrs_participant = models.BooleanField(default=False)
+    physical_locations = models.IntegerField(null=True)
```

Comment:
suggestion: it always feels odd to me that we make fields optional when actually we always expect a value. My preference is to set a default (e.g. zero). I know this means we technically have "wrong" values when we first deploy, but given we're in control of when the data is gathered separately from when it's displayed I don't think it's a major issue.